src/core/storybook/reservationListArgs.ts

```diff
@@ -0,0 +1,248 @@
+    defaultValue: "Others are queueing for this material",
+    control: { type: "text" }
+  },
+  reservationDetailsExpiresTitleText: {
+    defaultValue: "Pickup deadline",
+    control: { type: "text" }
+  },
+  reservationDetailsDigitalMaterialExpiresTitleText: {
+    defaultValue: "Borrow before",
+    control: { type: "text" }
+  },
+  reservationDetailsExpiresText: {
+    defaultValue: "Your reservation expires @date!",
+    control: { type: "text" }
+  },
+  reservationDetailsSaveText: {
+    defaultValue: "Save",
+    control: { type: "text" }
+  },
+  reservationDetailsCancelText: {
+    defaultValue: "Cancel",
+    control: { type: "text" }
+  },
+  reservationListPauseReservationText: {
+    defaultValue: "Pause your reservations",
+    control: { type: "text" }
+  },
+  reservationListOnHoldAriaText: {
+    defaultValue: "Reservations have been paused in the following time span: ",
+    control: { type: "text" }
+  },
+  reservationListPauseReservationAriaModalText: {
+    defaultValue:
+      "This button opens a modal that covers the entire page and contains the possibility to pause physical reservations",
+    control: { type: "text" }
+  },
+  pauseReservationModalAriaDescriptionText: {
+    defaultValue:
+      "This modal makes it possible to pause your physical reservations",
+    control: { type: "text" }
+  },
+  pauseReservationModalHeaderText: {
+    defaultValue: "Pause reservations on physical items",
+    control: { type: "text" }
+  },
+  pauseReservationModalBodyText: {
+    defaultValue:
+      "Pause your reservations early, since reservations that are already being processed, will not be paused.",
+    control: { type: "text" }
+  },
+  pauseReservationModalCloseModalText: {
+    defaultValue: "Close pause reservations modal",
+    control: { type: "text" }
+  },
+  dateInputsStartDateLabelText: {
+    defaultValue: "Start date",
+    control: { type: "text" }
+  },
+  dateInputsEndDateLabelText: {
+    defaultValue: "End date",
+    control: { type: "text" }
+  },
+  pauseReservationModalBelowInputsTextText: {
+    defaultValue: "",
+    control: { type: "text" }
+  },
+  pauseReservationModalLinkText: {
+    defaultValue:
+      "Read more about pausing reservertions and what that means here",
+    control: { type: "text" }
+  },
+  pauseReservationModalSaveButtonLabelText: {
+    defaultValue: "Save",
+    control: { type: "text" }
+  },
+  listDetailsNothingSelectedLabelText: {
+    defaultValue: "Pick",
+    control: { type: "text" }
+  },
+  showMoreText: {
+    name: "Show more Text",
+    defaultValue: "show more",
+    control: { type: "text" }
+  },
+  resultPagerStatusText: {
+    name: "Result pager status text",
+    defaultValue: "Showing @itemsShown out of @hitcount results",
+    control: { type: "text" }
+  }
+};
+
+export interface ReservationMaterialDetailsProps {
+  reservationDetailsRemoveDigitalReservationText: string;
+  reservationDetailsDateOfReservationTitleText: string;
```

Comment:
Consider if this interface is necessary.

I wonder what this exported interface is used for. It does not seem to be used anywhere at the moment (I am reading this commit by commit).

Reply:
Having looked over the entire PR it might be some copy/pasting from another `storybook/Args.ts` file.

Reply:
👍